src/components/Products/Products.jsx

```diff
@@ -5,6 +5,8 @@ import { useProduct } from "../../context/product-context";
 import { filterByCategory } from "../../SortingAndFiltering/filterByCategory";
 
 import { sortByPrice } from "../../SortingAndFiltering/sortByPrice";
+import { sortByRating } from "../../SortingAndFiltering/sortByRating";
+import {filterByPriceRange} from '../../SortingAndFiltering/filterByPriceRange';
```

Comment:
Suggestion: can use barrel export and import directly from `./sortingAndFiltering`

Reply:
done